src/Sylius/Bundle/ApiBundle/Resources/config/serialization/Order.xml

```diff
@@ -110,6 +110,7 @@
 
         <attribute name="localeCode">
             <group>shop:order:account:read</group>
+            <group>order:read</group>
```

Comment:
Why do we need it here?

Reply:
The steps I have implemented or refactored in this PR are now calling for completed order with GET orders/id. Because there is no localeCode in them I have added this serialization group in here. Well to be honest this is not part of PR ( as title says) but now the steps are much better ( not performance wise as I need to make a one call per step more), but we have current state of the cart, not state from the step before.
Also I think this group should be here ( so we can see what locale is added to the cart/order).
Well You are right that this should not be here, but otherwise I would need to change step `And my order's locale should be "English (United States)"` and when we add this serialization in other PR we would need to make a refactor of it again.

Reply:
I think we should go with it here, If we make another PR for this one serialization change, it will give us more work in future 😢